src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -701,21 +818,35 @@ def predict(self, data, verbose = True):
             Rows: 10
             [3, 4, 3, 3, 4, 5, 8, 8, 8, 4]
         """
+        output_type = output_type.lower()
```

Comment:
This seems like kind of an odd choice. I could be wrong but I don't think this is something that other toolkits do. I think this parameter should be case sensitive.

Reply:
Done.